### PR TITLE
Add watch task

### PR DESCRIPTION
**Pull request by tofumatt**
r? @sole or @brittanystoroz 

Instant feedback when coding!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,5 +21,8 @@
     "gulp-nodeunit": "0.0.5",
     "gulp-replace": "^0.5.0",
     "jshint-stylish": "^1.0.0"
+  },
+  "devDependencies": {
```
Comment (tofumatt) [inline package.json]: Oops! This shouldn't be a `devDependency`!

**Comment by sole** [comment]
Looks good, the travis task is failing, though, very ironic! :D

**Comment by sole** [comment]
You should also bump the package.json version so the module can be published to npm and we upgrade all other modules using it :)

**Comment by tofumatt** [comment]
Oh, damn, I'll fix that.

Also I bumped it in another PR but I'll do that here when I fix the tests.

**Pull request merged**